provider/backup_utils.py

```diff
@@ -405,6 +405,21 @@ def create_image_by_params(vm, params, image_name):
     return image
 
 
+def create_image_with_data_file(vm, params, image_name):
+    """Create image with data_file"""
+    image_list = []
+    image_params = params.object_params(image_name)
+    data_file_tag = image_params.get("data_file")
```

Comment:
suggest to use image_data_file and image_data_file_raw

Reply:
@zhencliu, update it in cfg, ex, image_data_file_sn2 = sn1, please help to check again.